docs/standards/VEP/vep-1155.md

```diff
@@ -0,0 +1,538 @@
+}    
+```
+
+## Rationale
+
+### Metadata Choices
+
+The `symbol()` function (found in the [TIP-3] standard) was not included as we do not believe this is a globally useful piece of data to identify a generic virtual item / asset and are also prone to collisions. Short-hand symbols are used in tickers and currency trading, but they aren’t as useful outside of that space.
+
+The `name()` function (for human-readable asset names, on-chain) was removed from the standard to allow the Metadata JSON to be the definitive asset name and reduce duplication of data. This also allows localization for names.
+
+The `decimals()` function was removed from the standard since the user can't own only fraction of associated data in json. In practice, we can only set it to zero. There is not need to return obvious value.
+
+Since someone should pay for storage fee the decision was made to store json metdata in MultiTokenWallet contract owned by each user.
+
+### MultiTokenWallet destroy
+
+It's possible to destroy MultiTokenWallet or/and indexes contracts when balance is zero. However it results in excessive complexity and additional Venom expenses. We can't handle onBounce if contract is destroyed during transfer. Therefore we need some mechnism to singal back that transfer is completed successfully. This will add no only complexity but additional fee. Besides there are many cases where zero balance is temporary (for example put on sale and cancel sale). As a result it makes sense to have an optional destroy and declare that the owner should decide whether destroy is needed or not.
+
+### Reuse interfaces from [TIP-4.2], [TIP-4.3]
+
+`TIP4_2JSON_Metadata` interface from [TIP-4.2] and `TIP4_3NFT` interface from [TIP-4.3] standard are used as is for MultiTokenWallet. These interfaces can be used as is and cover needed functinality. However some names can be in doubt (for example `nft` in `getInfo`). There is no reason to introduce new interfaces just to have better naming.
+
+### Indexes
+
+`Index` contracts from [TIP-4.3] can help us to find MultiTokenWallet contracts as well as NFT. However they are not completely interchangeable. In order to find MultiTokenWallet only stamp is changed from "nft" to "fungible".
+
+## Visualization
+
+### Legend
+
+![Legend](assets/VEP-1155/legend.svg)
+
+### Example 1: Put on sell by [TIP-3] tokens
+
+![Put on sell by TIP-3 tokens](assets/VEP-1155/putonsell.png)
+
+### Example 2: Buy by [TIP-3] tokens
+
+![Buy by TIP-3 tokens](assets/VEP-1155/buy.png)
+
+## Backwards compatibility
+
+This standard is compatible with [TIP-4.1], [TIP-4.2], [TIP-4.3] standards.
+
+## Security considerations
+
+There are no security considerations related directly to the implementation of this standard.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](/standards/VEP/LICENSE.md).
+
+## References
+
+- [TIP-3]
+- [TIP-4.1]
+- [TIP-4.2]
+- [TIP-4.3]
+- [TIP-6.1]
+- [Ethereum EIP-1155]
+
+[TIP-3]: ../TIP/TIP-3/core-description.md
```

Comment:
Please use a full link here, not a relative. Like https://docs.venom.foundation/standards/TIP/TIP-3/core-description
Cuz VEPs shouldn't be depended of docs itself in this way